content/en/news/releases/1.18.x/announcing-1.18/change-notes/index.md

```diff
@@ -0,0 +1,381 @@
+
+- **Fixed** WorkloadEntry resources never being cleaned up if multiple
+WorkloadEntries were auto-registered with the same IP and network.
+  ([Issue #43950](https://github.com/istio/istio/issues/43950))
+
+- **Fixed** The `dns_upstream_failures_total` metric was mistakenly deleted in the previous release.
+  ([Issue #44151](https://github.com/istio/istio/issues/44151))
+
+- **Fixed** an issue where ServiceEntry and Service had undefined or empty workload selectors. If the workload selector is undefined or empty, 
+ServiceEntry and Service should not select any WorkloadEntry or endpoint.
+  
+
+- **Fixed** An issue where a Service Entry configured with partial wildcard hosts generates a warning during validation as the config can some times generate invalid server name match.  ([Issue #44195](https://github.com/istio/istio/issues/44195))
+
+- **Fixed** an issue where `Istio Gateway` (Envoy) would crash due to a duplicate `istio_authn` network filter in the Envoy filter chain.
+  ([Issue #44385](https://github.com/istio/istio/issues/44385))
+
+- **Fixed** a bug where services are missing in gateways if PILOT_FILTER_GATEWAY_CLUSTER_CONFIG is enabled.  ([Issue #44439](https://github.com/istio/istio/issues/44439))
+
+- **Fixed** cpu usage abnormally high when cert specified by DestinationRule are invalid.
+  ([Issue #44986](https://github.com/istio/istio/issues/44986))
+
+- **Fixed** an issue where changing a label on a workload instance with a previously matched `ServiceEntry` would not properly get removed.
+  ([Issue #42921](https://github.com/istio/istio/issues/42921))
+
+- **Fixed** istiod not reconciling k8s gateway deployments and services when they are changed.
+  ([Issue #43332](https://github.com/istio/istio/issues/43332))
+
+- **Fixed** an issue where istiod does not retry resolving east-west gateway hostnames on failure.
+  ([Issue #44155](https://github.com/istio/istio/issues/44155))
+
+- **Fixed** an issue where istiod generates incorrect endpoints when it fails to resolve east-west gateway hostnames.
+  ([Issue #44155](https://github.com/istio/istio/issues/44155))
+
+- **Fixed** an issue where sidecars do not proxy DNS properly for a hostname backed by multiple services.
+  ([Issue #43152](https://github.com/istio/istio/pull/43152))
+
+- **Fixed** an issue where updating Service ExternalName does not take effect.
+  ([Issue #43440](https://github.com/istio/istio/issues/43440))
+
+- **Fixed** an issue causing VMs using auto-registration to ignore labels other than those defined in a WorkloadGroup.
+  ([Issue #32210](https://github.com/istio/istio/issues/32210))
+
+- **Upgraded** the gateway-api integration to read `v1beta1` resources for `ReferenceGrant`, `Gateway`, and `GatewayClass`. Users of the gateway-api must be on v0.6.0+ before upgrading Istio. `istioctl x precheck` can detect this issue before upgrading.
+  
+
+- **Removed** support for `proxy.istio.io/config` annotation applied to Kubernetes `Gateway` pods.
+  
```

Comment:
most of the places have a single line.. may be it makes sense to keep this uniform ? i think all similar entities are grouped togther, and then we have two lines to the next group. But that style is not followed throughout. So better to do the same

Reply:
@kfaseela It doesn't seem to matter, from a quick play locally it looks like the website does some kind of reordering and grouping at build time, so there's no particular need to worry about the ordering of the updated, improved, added etc items, or the spacing as it makes no difference to the presentation

Reply:
The linter cares about using a single line for spacing.